Web: Show a "Give feedback" banner and link to a survey (#13125)
laurent22/joplin: readme/privacy.md

| OCR | If optical character recognition is enabled on desktop, the application will download the language files from https://cdn.jsdelivr.net/npm/@tesseract.js-data/. | Enabled | Yes 
| Crash reports | If you have enabled crash auto-upload, the application will upload the report to Sentry when a crash happens. When Sentry is initialised it will also connect to `sentry.io`. | Disabled | Yes
| Handwriting recognition | If the 'handwriting recognition' setting is enabled, users can send images to Joplin Server/Cloud to be transcribed. Only images selected with the 'Recognize handwritten image' are affected. | Disabled | Yes

> | Beta feedback | (**Web app only**) If the feedback dialog is used, Joplin contacts `objects.joplinusercontent.com` to submit the response. | Enabled | Yes

Author: Or maybe
```suggestion
| Beta feedback | On web, Joplin sends responses to the beta feedback survey to  `objects.joplinusercontent.com`. | Enabled | Yes
```
